spec/mercure.md

```diff
@@ -46,7 +46,7 @@ The keywords **MUST**, **MUST NOT**, **REQUIRED**, **SHALL**, **SHALL NOT**, **S
 ![Discovery Schema](discovery.png)
 
 If the publisher is a server, it **SHOULD** advertise the URL of one or more hubs to the subscriber, allowing it to receive live updates when topics are updated.
-If more than one hub URL is specified, it is expected that the publisher will notify each hub, so the subscriber **MAY** subscribe to one or more of them.
```

Comment:
This sentence also comes from the WebSub spec:

> The URL of one or more hubs designated by the publisher. If more than one hub URL is specified, it is expected that the publisher notifies each hub, so the subscriber may subscribe to one or more of them.

May can we keep the wording from the W3C?